services/provider/server/server.go

```diff
@@ -799,6 +803,23 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 				rbdStorageClassData["encryptionKMSID"] = storageRequest.Spec.EncryptionMethod
 			}
 
+			blockPool := &rookCephv1.CephBlockPool{}
+			err = s.client.Get(ctx, types.NamespacedName{Name: rns.Spec.BlockPoolName, Namespace: s.namespace}, blockPool)
+			if err != nil {
+				return nil, status.Errorf(codes.Internal, "failed to get %s CephBlockPool. %v", blockPool.Name, err)
+			}
+
+			// SID for RamenDR
+			storageID := util.CalculateMD5Hash(struct {
+				CephClusterID      string `json:"cephClusterID"`
+				BlockPoolID        string `json:"blockPoolID"`
+				RadosNamespaceName string `json:"radosNamespaceName"`
+			}{
+				cephCluster.Status.CephStatus.FSID,
+				fmt.Sprintf("%v", blockPool.Status.PoolID),
+				rns.Name,
+			})
```

Comment:
Can we have CalculateStorageId as its own function/util? If not then the usage of an anonymous struct is problamatic as it require the redefinition of the struct on each call site